gen3/neural-networks/basic-example/utils/arguments.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+from typing import Tuple
+import os
+
+
+def initialize_argparser():
+    value = os.getenv("DEPTHAI_DEVICE_NAME_LIST")
+    value = value if value else ""
+    
+    """Initialize the argument parser for the script."""
+    parser = argparse.ArgumentParser()
+    parser.description = "General example script to run any model available in HubAI on DepthAI device. \
+        All you need is a model slug of the model and the script will download the model from HubAI and create \
+        the whole pipeline with visualizations. You also need a DepthAI device connected to your computer. \
+        Currently, only RVC2 devices are supported. If using OAK-D Lite, please set the FPS limit to 28."
+
+    parser.add_argument(
+        "-s",
+        "--model_slug",
```

Comment:
Let's call this "model" instead of "model_slug" from now on. So this would then be: -m, --model_slug
Also update the help string here and in the parser.description to remove "slug" mentioning